dev-libs/pigpio/files/pigpiod.initd

```diff
@@ -0,0 +1,7 @@
+#!/sbin/openrc-run
```

Comment:
is it really an init script? It defines only two variables, does not seem like it runs something.

Reply:
Well, it should start and stop the binary using the openrc framework. There needs to be a way to integrate this in the startup script (rc-update etc..), thus this "almost empty" init script. This is also present in some other existing packages...

Reply:
@Zlogene, you seem not to be up-to-date on OpenRC.